### PR TITLE
add custom agent scripts support

### PR DESCRIPTION
**Pull request by ksimon1**
Add support for custom scripts.
@bronhaim, @emesika please can you add your opinion?

### CODE REVIEW

**Comment by bronhaim** [comment]
@jfriesse didn't you start to work on custom agent for kubectl actions?

File: agent-job-image/Dockerfile [hunk 1]
```diff
@@ -19,5 +19,15 @@ RUN yum upgrade -y;
...
 
+RUN echo $'[kubernetes] \n\
+name=Kubernetes \n\
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64 \n\
+enabled=1 \n\
+gpgcheck=1 \n\
+repo_gpgcheck=1 \n\
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg' > /etc/yum.repos.d/kubernetes.repo
+
+RUN yum install -y kubectl
```
Comment (bronhaim) [inline agent-job-image/Dockerfile]: add ;yum clean all

File: agent-job-image/Dockerfile [hunk 2]
```diff
@@ -19,5 +19,15 @@ RUN yum upgrade -y;
 
 RUN yum install fence-agents* -y; yum clean all
 
+RUN echo $'[kubernetes] \n\
+name=Kubernetes \n\
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64 \n\
```
Comment (bronhaim) [inline agent-job-image/Dockerfile]: you changed the base image to f28, find the right repo here

**Comment by jfriesse** [comment]
@bronhaim Yep, I did, but never completed it :( Actually, when idea to move to ansible script appeared it didn't make too much sense (IMHO) to have agent which just calls `kubectl cordon|ncordon` (same result can be achieved in the Ansible script more easily).

**Review by mwperina — APPROVED** [review]
Can we merge?

**Comment by bronhaim** [comment]
mm.. we can. I just wondered if the work for kubectl agent was done or not, I think this approach is also fine but it requires verification, not sure the privileges you set are enough

**Pull request merged**